src/components/tables/LMTable/LMTable.vue

```diff
@@ -45,11 +45,17 @@
         </div>
       </template>
       <template v-slot:poolNameCell="pool">
-        <div class="px-6 py-4">
+        <div class="px-6 py-4 items-center">
           <TokenPills
             :tokens="orderedPoolTokens(pool)"
             :isStablePool="isStableLike(pool.poolType)"
           />
+          <div
+            v-if="pool.dynamic.isNewPool"
+            class="rounded bg-red-500 text-xs px-1 shadow-sm text-white uppercase ml-2"
```

Comment:
@garethfuller @1saf  the class repeats itself on the PoolsTable, and I was wondering - should this become a component? or perhaps we make a new shared global css file? (to me it feels like its just quicker to have this as a reusable css class)

For example, I could name it: `.badge-red` and reuse that. (and it will reside in `components.css`)